outputs/policyreport.go

```diff
@@ -67,12 +67,12 @@ func NewPolicyReportClient(config *types.Configuration, stats *types.Statistics,
 	}, nil
 }
 
-//make one more input ns string
+//to do n+1 logic
 // PolicyReportPost creates Policy Report Resource in Kubernetes
 func (c *Client) PolicyReportCreate(falcopayload types.FalcoPayload) {
 	warnbound = c.Config.PolicyReport.Warning
-	r, count := newResult(falcopayload)
-	if count == 0 {
+	r, namespaceScoped := newResult(falcopayload)
+	if namespaceScoped == true {
 		policyr := c.Crdclient.Wgpolicyk8sV1alpha2().PolicyReports("default")
```

Comment:
We should consider to allow users to set the namespace they want for reports, with `falco` as default, or we create reports in same namespace than the resource in the event, wdyt?

Reply:
Hey @Issif, so what I had in mind was that we will have N namespace specific policyreports in their own namespaces and 1 clusterwide report for non namespace specific reports. PolicyReport output would generate all of them and the user could decide which one the user wants to use.
I believe this is along the lines of the second approach that you suggested. Would love to know your thoughts on this!

Reply:
Hey @Issif, so what I had in mind was that we will have N namespace specific policyreports in their own namespaces and 1 clusterwide report for non namespace specific reports. PolicyReport output would generate all of them and the user could decide which one the user wants to use.
I believe this is along the lines of the second approach that you suggested. Would love to know your thoughts on this!

Reply:
Sure @Issif I will work on this!